include/cppaikit/Config.hpp.in

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+constexpr auto CppAIKit_VERSION_MAJOR = ${CppAIKit_VERSION_MAJOR};
```

Comment:
Observation:  this should probably/definitely go inside a proper namespace, of course